apps/src/applab/sanitizeHtml.js

```diff
@@ -64,17 +64,19 @@ function warnAboutUnsafeHtml(warn, unsafe, safe) {
  *     any unsafe html was removed from the output.
  */
 module.exports = function sanitizeHtml(unsafe, warn) {
+  var defaultAttributes = ['id', 'class', 'height', 'width', 'style'];
+
   var safe = sanitize(unsafe, {
     allowedTags: sanitize.defaults.allowedTags.concat([
       'button', 'canvas', 'img', 'input', 'option', 'label', 'select']),
     allowedAttributes: $.extend({}, sanitize.defaults.allowedAttributes, {
-      button: ['id', 'class', 'style', 'data-canonical-image-url'],
-      canvas: ['id', 'class', 'style', 'width', 'height'],
-      div: ['id', 'class', 'style', 'data-canonical-image-url', 'contenteditable', 'tabindex', 'xmlns'],
-      img: ['id', 'class', 'data-canonical-image-url', 'src', 'style'],
-      input: ['id', 'checked', 'class', 'max', 'min', 'placeholder', 'step', 'style', 'type', 'value'],
-      label: ['id', 'class', 'style'],
-      select: ['id', 'class', 'style']
+      button: defaultAttributes.concat(['data-canonical-image-url']),
+      canvas: defaultAttributes,
+      div: defaultAttributes.concat(['contenteditable', 'data-canonical-image-url', 'tabindex', 'xmlns']),
+      img: defaultAttributes.concat(['data-canonical-image-url', 'src']),
+      input: defaultAttributes.concat(['checked', 'max', 'min', 'placeholder', 'step', 'type', 'value']),
+      label: defaultAttributes,
+      select: defaultAttributes
```

Comment:
If we were using ES6, you could just do stuff like:
```
button: [...defaultAttributes, 'data-canonical-image-url']
```
Look forward to being in that world :)